Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamerBase.cpp

```diff
@@ -1568,15 +1568,24 @@ void MediaPlayerPrivateGStreamerBase::emitPlayReadySession(PlayreadySession* ses
 #endif // USE(PLAYREADY)
 
 #if USE(OPENCDM)
-void MediaPlayerPrivateGStreamerBase::emitOpenCDMSession()
+CDMSessionOpenCDM* MediaPlayerPrivateGStreamerBase::openCDMSession()
 {
     if (!m_cdmSession)
-        return;
+        return nullptr;
+
 #if ENABLE(LEGACY_ENCRYPTED_MEDIA)
-    CDMSessionOpenCDM* cdmSession = static_cast<CDMSessionOpenCDM*>(m_cdmSession);
+    return static_cast<CDMSessionOpenCDM*>(m_cdmSession);
```

Comment:
We should do this a downcast but it involves boilerplate and I won't bother about this now unless we get nasty crashes.